addons/website_sale/static/src/css/website_sale.css

```diff
@@ -108,21 +108,26 @@
   right: 0;
   bottom: 0;
   overflow: hidden;
-  padding: 0 15px 24px 10px;
+  padding: 0 15px 0px 10px;
   min-height: 56px;
   border-top: 1px solid rgba(255, 255, 255, 0.2);
   background: rgba(255, 255, 255, 0.75);
   z-index: 5;
 }
+.oe_product section  h5 {
+  line-height: 1.2;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
I would not change the title margins and line height as it is customized by theme and changing this changes the standard design. Remove this and remove the padding-top of `.oe_product .product_price`. That should have around the same effect but with proper spacing on top of the title, and without a new rule.